app/cypress/e2e/assessment-units.cy.ts

```diff
@@ -11,305 +11,128 @@ describe("Data Profile Assessment Units", () => {
-        expect($elem.find("span").first().text().trim()).equal(
-          `${origin}/attains/data/assessmentUnits?format=csv`
-        );
-      });
-  });
-
-  it("Verify copy box Assessment Units API Query text flavor 2", () => {
-    cy.findByRole("checkbox", { name: "Retired" }).click({ force: true });
-
-    cy.selectOption("input-locationText", "adams county");
+    cy.selectOption("input-locationText", "buffalo county");
 
+    //Assessment Unit ID
     cy.selectOption("input-assessmentUnitId", "AL-Gulf-of-Mexico-2");
 
-    cy.findByTestId("api-query-copy-box-container")
-      .should("exist")
-      .should(($elem) => {
-        expect($elem.find("span").first().text().trim()).equal(
-          `${origin}/attains/data/assessmentUnits?format=csv&assessmentUnitId=AL-Gulf-of-Mexico-2&assessmentUnitState=R&locationText=Adams County`
-        );
-      });
-  });
-
-  it("Verify copy box Assessment Units API Query text flavor 3", () => {
-    cy.findByRole("checkbox", { name: "Historical" }).click({ force: true });
-
-    cy.selectOption("input-organizationName", "wisconsin");
-
-    cy.selectOption("input-waterType", "ocean");
-
-    cy.findByText("Tab-separated (TSV)").click();
-
-    cy.findByTestId("api-query-copy-box-container")
-      .should("exist")
-      .should(($elem) => {
-        expect($elem.find("span").first().text().trim()).equal(
-          `${origin}/attains/data/assessmentUnits?format=tsv&assessmentUnitState=H&organizationName=Wisconsin&waterType=OCEAN`
-        );
-      });
-  });
-
-  it("Verify copy box Assessment Units API Query text flavor 4", () => {
-    cy.findByRole("checkbox", { name: "Active" }).click({ force: true });
-
-    cy.selectOption("input-organizationId", "sfnoes");
-
-    cy.selectOption("input-state", "texas");
-
-    cy.selectOption("input-useClassName", "vii");
-
-    cy.findByText("JavaScript Object Notation (JSON)").click();
-
-    cy.findByTestId("api-query-copy-box-container")
-      .should("exist")
-      .should(($elem) => {
-        expect($elem.find("span").first().text().trim()).equal(
-          `${origin}/attains/data/assessmentUnits?format=json&assessmentUnitState=A&organizationId=SFNOES&state=TX&useClassName=VII`
-        );
-      });
-  });
-
-  it("Verify copy box Assessment Units API Query text flavor 5", () => {
+    const queryValue =
+      "format=csv&assessmentUnitId=AL-Gulf-of-Mexico-2&assessmentUnitState=R&locationText=Buffalo%20County";
+
+    cy.selectCopyBox(
+      "current-query-copy-box-container",
+      `${origin}/attains/assessmentUnits/#${queryValue}`
+    );
+    cy.selectCopyBox(
+      "api-query-copy-box-container",
+      `${origin}/attains/data/assessmentUnits?${queryValue}`
+    );
+    cy.selectCopyBox(
+      "curl-copy-box-container",
+      `curl -X POST --json ${JSON.stringify(
+        '{"filters":{"assessmentUnitId":["AL-Gulf-of-Mexico-2"],"assessmentUnitState":["R"],"locationText":["Buffalo County"]},"options":{"format":"csv"}}'
```

Comment:
```suggestion
        '{"filters":{"assessmentUnitId":["AL-Gulf-of-Mexico-2"],"assessmentUnitStatus":["R"],"locationText":["Buffalo County"]},"options":{"format":"csv"}}'
```

The `assessmentUnitState` column is being renamed to `assessmentUnitStatus` in the database, so this needs to be updated everywhere. I'll go ahead and push a commit up for this.

NOTE: It will break the tests until we get the ETL changes merged in and you re-run the ETL process.